### ISSUE TITLE
Unable to parse json from int64

### ISSUE THREAD

**Issue by ibnuda** [fable-compiler/Fable#1104]
### Description

In a type which has int64 as one of its properties, `ofJson` unable to parse it correctly by translate it to `low`, `high`, and `signed`.

### Repro code
https://github.com/ibnuda/Jaran/blob/5df8b3523fa8188db99637501759cf90730293a3/src/App.fs#L16
and
https://github.com/ibnuda/Jaran/commit/07b509de67d7432dbfea5d14dbb5ed4c5e2da396#diff-2e614e41bedf555adac621bac769e0e6R17
### Expected and actual results
expected : should be an instance of `Something` with properties `lastuser = Administrator` and `lastactive = 636372711333291008`
### Related information
'
![err](https://user-images.githubusercontent.com/8169908/28971993-403a822a-7958-11e7-963f-b24eaa05c445.PNG)

* Fable version (`dotnet fable --version`): 1.1.17
* Operating system: Windows 10

**Comment by rkosafo** [comment]
Error exists also in `0.7`. For a workaround, I often use int64 on the server side and int in fable.

**Comment by ibnuda** [comment]
Oh God.... It works.
![oh god it works](https://user-images.githubusercontent.com/8169908/28972931-cd070ff4-795b-11e7-9da5-9dae88e96f2b.PNG)
I think it's a good idea to explicitly mention it on the documentation.

**Comment by Zaid-Ajaj** [comment]
@ibnuda You could stil use `int64` on the client if you "manually" parse the json (workaround for now)

```fs
open Fable
open Fable.Core
open Fable.Core.JsInterop

[<Emit("$1[$0]")>]
let getProp<'t> (key: string) (json: obj) : 't = jsNative

[<Emit("JSON.parse($0)")>]
let jsonParse (json: string) : obj option = jsNative

type Something =
    { lastuser : string
      lastactive : int64 }
      
      
let somethingOfJson (json: string) : Something option = 
    match jsonParse json with
    | Some value -> 
        let lastuser = getProp<string> "lastuser" value
        let lastActiveStr = getProp<string> "lastactive" value
        { lastuser = lastuser; lastactive = int64 lastActiveStr }
        |> Some
    | None -> None
```
use it like so:
```fs
let u = """ { "lastuser": "Administrator", "lastactive":"636372711333291008" } """ // see the quotes
somethingOfJson u
|> printf "this : %A"
// "this : {\"lastuser\":\"Administrator\",\"lastactive\":\"636372711333291008\"}"
```

**Comment by alfonsogarciacaro** [comment]
Hi @ibnuda, thanks a lot for your report! The problem should be fixed now, but it'll be necessary to update both Fable.Core and Fable.JsonConverter. I'll try to publish new versions of the packages as soon as possible :+1:

**Comment by rkosafo** [comment]
@alfonsogarciacaro Will the update be for both 0.7 and 1.0?

**Comment by alfonsogarciacaro** [comment]
@rkosafo 0.7 has been updated for a while now :/ I'd recommend going for 1.1.x as you'll take advantage of many other bug fixes. Is it not possible to upgrade your app?

**Comment by rkosafo** [comment]
@alfonsogarciacaro Tried moving to v1 for the two most recent projects. Beyond helloworld, I faced just too many problems with the .net core - no vs support, transient dependency and some strange new errors at every turn :(. After some days with very little success, I decided to stay away from .net core altogether for now and use 0.7 for the new projects. I'm following it closely though. I'll try it again when the current projects are complete and then upgrade.
